config/samples/core_v1alpha1_account.yaml

```diff
@@ -1,19 +0,0 @@
-apiVersion: core.openmfp.io/v1alpha1
```

Comment:
why remove the example? I guess changing it to a resource that is already available in kubernetes like a ConfigMap would be better

Reply:
My idea was to rely on the resources that are defined in the local kind setup with the openmfp sample. 
We soon also have the fga connection and dependencies on the schema there. So a single account resource will anyways not be sufficient for any meaningful test

Reply:
ok i changed the sample to work with a config map, seems to work that way